frontend/src/styles/element.scss

```diff
@@ -253,3 +258,8 @@ html {
 .logo {
     color: var(--el-color-primary);
 }
+
+.custom-input-textarea {
+    background-color: #f5f7fa !important;
+    color: var(--el-color-info) !important;
+}
```

Comment:
There is no apparent issue, irregularity, or optimization suggestion in this text regarding the code. It appears to be valid HTML/CSS and follows established standards. No modifications are requested.